src/components/MainSection/MainSection.tsx

```diff
@@ -1,6 +1,7 @@
-import './MainSection.css';
+import "./MainSection.css";
+import React from "react";
 
-function MainSection({ children }) {
+function MainSection({ children }: Children) {
```

Comment:
```suggestion
function MainSection({ children }: PropsWithChildren) {
```
React 내에 PropsWithChildren 같은 타입도 제공하고 있어서 굳이 children을 직접 선언해주지 않아도 괜찮습니다~ 비슷한 계열의 타입들이 많아서 잘 활용해보시면 좋을 거 같아요!